File: rating-quarkus-java/src/main/java/com/senacor/vienna/qk/rating/RatingResource.java
package com.senacor.vienna.qk.rating;

import javax.ws.rs.GET;
import javax.ws.rs.Path;
import javax.ws.rs.Produces;
import javax.ws.rs.core.MediaType;

@Path("/C:/Progs/git-2.17.0-64/rating")
public class RatingResource {

    @GET
    @Produces(MediaType.TEXT_PLAIN)
    public String hello() {
        return "hello";
    }
}

File: rating-quarkus-kotlin/src/main/kotlin/com/senacor/vienna/qk/rating/RatingResource.kt
package com.senacor.vienna.qk.rating

import javax.ws.rs.GET
import javax.ws.rs.Path
import javax.ws.rs.Produces
import javax.ws.rs.core.MediaType

@Path("/C:/Progs/git-2.17.0-64/rating")
class RatingResource {

    @GET
    @Produces(MediaType.TEXT_PLAIN)
    fun hello() = "hello"
}